src/main/java/com/nerodesk/om/aws/AwsDoc.java

```diff
@@ -120,6 +120,11 @@ public void read(@NotNull final OutputStream output) throws IOException {
     @Override
     public void write(final InputStream input, final long size)
         throws IOException {
+        if (size == 0) {
```

Comment:
@longtimeago  if we decide to save doc with zero length we should remove SafeDoc and implement the zero-length file storage...
right now if we remove SafeDoc, the Doc instance will be broken on the zero length file